bundles/org.openhab.binding.freeathomesystem/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="freeathomesystem" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>FreeAtHomeSystem Binding</name>
+	<description>This is the binding for Free@Home system.</description>
+
+</binding:binding>
```

Comment:
Please convert to the new `addon.xml`.

Reply:
Corrosponding issue is here: https://github.com/openhab/openhab-core/issues/2058

Reply:
The following files had format violations:
[ERROR] src/main/resources/OH-INF/addon/addon.xml

Please run spotless

Reply:
spotless done
Thank you the trigger @lsiepel

Reply:
correction as proposed done